trulens_eval/trulens_eval/instruments.py

```diff
@@ -743,6 +743,13 @@ def instrument_object(
         # https://github.com/pydantic/pydantic/blob/11079e7e9c458c610860a5776dc398a4764d538d/pydantic/main.py#LL370C13-L370C13
         # .
 
+        # Recursively instrument inner components
```

Comment:
This should already be done by what is on line 835. Do you have example of that not recurring in correctly? If so, can you fix it there?

Reply:
The classes are already included in instrumentation list in tru_llama. What do you mean by "fix it there"?

Reply:
I mean the subcomponent instrumentation is done around line 835 so I was suggesting that your fix to it not correctly recurring down should be put in the same place.

Reply:
L835 seems to cover unrecognized container types? I don't think that's what's happening here

Reply:
Would like to merge this soon, got customer ask for this today.